### ISSUE TITLE
[PATCH] perlre.pod  buglet

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#3308]
Migrated from [rt.perl.org#5540](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=5540) (status was 'resolved')

Searchable as RT5540$

**Comment by p5pRT** [comment]
### From root@ak-75.mind.de


perlre\.pod has a misquoted piece that gets rendered as

&nbsp;    ""\<"\<b>\<>\<a>\<>\<r>\<>">

instead of

&nbsp;    "\<>\<b>\<>\<a>\<>\<r>\<>"

The following patch fixes this&#8203;:


<details><summary>Inline Patch</summary>

```diff
--- /usr/sources/perl/repoperls/perl-5.7.0@8573/pod/perlre.pod@5873	Sun Jan 28 20:26:44 2001
+++ /usr/sources/perl/repoperls/perl-5.7.0@8573/pod/perlre.pod	Sun Jan 28 20:27:05 2001
@@ -1096,7 +1096,7 @@
     $_ = 'bar';
     s/\w??/<$&>/g;
 
-results in C<"<><b><><a><><r><>">.  At each position of the string the best
+results in C<< <><b><><a><><r><> >>.  At each position of the string the best
 match given by non-greedy C<??> is the zero-length match, and the I<second 
 best> match is what is matched by C<\w>.  Thus zero-length matches
 alternate with one-character-long matches.
```
</details>


<details><summary>Perl Info</summary>

```


This perlbug was built using Perl 5.00503 - Wed Feb  2 15:34:50 EST 2000
It is being executed now by  Perl 5.007 - Sun Dec 10 14:23:48 CET 2000.

Site configuration information for perl 5.007:

Configured by root at Sun Dec 10 14:23:48 CET 2000.

Summary of my perl5 (revision 5.0 version 7 subversion 0) configuration:
  Platform:
    osname=linux, osvers=2.2.16a, archname=i686-linux
    uname='linux ak-71.mind.de 2.2.16a #1 smp wed jun 14 11:59:53 cest 2000 i686 unknown '
    config_args='-Dprefix=/usr/local/perl-5.7.0@7977 -Doptimize=-g -des -Dusedevel'
    hint=recommended, useposix=true, d_sigaction=define
    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
    useperlio=undef d_sfio=undef uselargefiles=define usesocks=undef
    use64bitint=undef use64bitall=undef uselongdouble=undef
  Compiler:
    cc='cc', ccflags ='-DDEBUGGING -fno-strict-aliasing -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64',
    optimize='-g',
    cppflags='-DDEBUGGING -fno-strict-aliasing -I/usr/local/include'
    ccversion='', gccversion='egcs-2.91.66 19990314/Linux (egcs-1.1.2 release)', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=1234
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=4, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib
    libs=-lnsl -lndbm -lgdbm -ldb -ldl -lm -lc -lposix -lcrypt -lutil
    perllibs=-lnsl -ldl -lm -lc -lposix -lcrypt -lutil
    libc=/lib/libc-2.1.3.so, so=so, useshrplib=false, libperl=libperl.a
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-rdynamic'
    cccdlflags='-fpic', lddlflags='-shared -L/usr/local/lib'

Locally applied patches:
    


@INC for perl 5.007:
    /usr/local/perl-5.7.0@7977/lib/5.7.0/i686-linux
    /usr/local/perl-5.7.0@7977/lib/5.7.0
    /usr/local/perl-5.7.0@7977/lib/site_perl/5.7.0/i686-linux
    /usr/local/perl-5.7.0@7977/lib/site_perl/5.7.0
    /usr/local/perl-5.7.0@7977/lib/site_perl
    .


Environment for perl 5.007:
    HOME=/root
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/X11R6/bin
    PERL_BADLANG (unset)
    SHELL=/bin/zsh

```
</details>

**Comment by p5pRT** [comment]
### From @jhi

> \-\-\- /usr/sources/perl/repoperls/perl\-5\.7\.0@&#8203;8573/pod/perlre\.pod@&#8203;5873    Sun Jan 28 20&#8203;:26&#8203;:44 2001
> \+\+\+ /usr/sources/perl/repoperls/perl\-5\.7\.0@&#8203;8573/pod/perlre\.pod    Sun Jan 28 20&#8203;:27&#8203;:05 2001

Thanks\, applied\.  \(Change \#8587\)
